### DISCUSSION TITLE
Vercel Deploy PAT_1 maxDuration 1-10 error

### DISCUSSION THREAD

**Discussion by AnthonyNet** [anuraghazra/github-readme-stats#2318] [Q&A] [answered]
I´ve clicked on the deploy button in the link/video and I´ve got this error.
The value for maxDuration must be between 1 second and 10 seconds, in order to increase this limit upgrade your plan.

https://camo.githubusercontent.com/5e471e99e8e022cf454693e38ec843036ec6301e27ee1e1fa10325b1cb720584/68747470733a2f2f76657263656c2e636f6d2f627574746f6e

https://youtu.be/n6d4KHSKqGk?t=228

![image](https://user-images.githubusercontent.com/67970914/206480948-e403df07-44a9-446e-ad3e-1d655b5e7623.png)

Any ideas where to find it? Thank you

**Comment by rickstaa** [comment, marked as answer]
@AnthonyZet thanks for your question. Please make sure you change the maxDuration parameter in the [vercel.json](https://github.com/anuraghazra/github-readme-stats/blob/master/vercel.json) file from 30 to 10 (see the warning admonition in https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-Vercel-instance for more information).
